src/library/Uno.Toolkit.Material/Styles/Controls/v2/TabBar.xaml

```diff
@@ -1,888 +1,1291 @@
-									<Storyboard x:Name="IndicatorTransitionHorizontalStoryboard">
-										<DoubleAnimation Storyboard.TargetName="IndicatorTransform"
-														 Storyboard.TargetProperty="TranslateX"
-														 From="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IndicatorTransitionFrom.X}"
-														 To="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IndicatorTransitionTo.X}"
-														 Duration="00:00:00.200" />
-									</Storyboard>
-								</VisualState>
-								<VisualState x:Name="Vertical">
-									<Storyboard x:Name="IndicatorTransitionVerticalStoryboard">
-										<DoubleAnimation Storyboard.TargetName="IndicatorTransform"
-														 Storyboard.TargetProperty="TranslateY"
-														 From="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IndicatorTransitionFrom.Y}"
-														 To="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IndicatorTransitionTo.Y}"
-														 Duration="00:00:00.200" />
-									</Storyboard>
-								</VisualState>
-							</VisualStateGroup>
-						</VisualStateManager.VisualStateGroups>
-						<ContentPresenter x:Name="IndicatorPresenter"
-										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-										  Content="{TemplateBinding Content}"
-										  ContentTemplate="{TemplateBinding ContentTemplate}"
-										  RenderTransformOrigin=".5,.5"
-										  Height="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IndicatorMaxSize.Height}"
-										  Width="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IndicatorMaxSize.Width}">
-							<ContentPresenter.RenderTransform>
-								<CompositeTransform x:Name="IndicatorTransform" />
-							</ContentPresenter.RenderTransform>
-						</ContentPresenter>
-					</Grid>
-				</ControlTemplate>
-			</Setter.Value>
-		</Setter>
-	</Style>
-	<!--#endregion-->
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:mobile="http://uno.ui/mobile"
+    xmlns:not_mobile="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:not_win="http://uno.ui/not_win"
+    xmlns:todo="what should be done"
+    xmlns:toolkit="using:Uno.UI.Toolkit"
+    xmlns:um="using:Uno.Material"
+    xmlns:utu="using:Uno.Toolkit.UI"
+    mc:Ignorable="d mobile todo not_win">
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <!--  Base Navigation-style TabBar Resources  -->
+            <x:Double x:Key="MaterialNavigationTabBarWidthOrHeight">80</x:Double>
```

Comment:
You're gonna hate me, but I think we should take off the "Material" prefix for all of these as we may have TopTabBarStyle/BottomTabBarStyle/VerticalTabBarStyle be variants for other design systems too in the future so most of these resource keys could be reused between design systems

Reply:
Will there also be a NavigationTabBar variant ? @kazo0

Reply:
Yeah we should assume this'll all be shared across design systems